recipe/meta.yaml

```diff
@@ -26,22 +26,24 @@ build:
 
 requirements:
   build:
+    - cmake >=3.23.1,!=3.25.0
+    - ninja
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}
+    - sysroot_{{ target_platform }} 2.17
   host:
     - librmm ={{ version }}
     - python
     - pip
     - cython >=0.29,<0.30
-    - spdlog >=1.8.5,<2
```

Comment:
This seems to be [handled by `librmm`]( https://github.com/conda-forge/librmm-feedstock/blob/dca6148b45adc86f530a059d30aab16ade271833/recipe/meta.yaml#L33 ). So think we can drop it from here